packages/create-svelte/shared/+eslint+prettier+typescript/eslint.config.js

```diff
@@ -0,0 +1,33 @@
+import js from '@eslint/js';
+import ts from 'typescript-eslint';
+import svelte from 'eslint-plugin-svelte';
+import prettier from 'eslint-config-prettier';
+import globals from 'globals';
+
+/** @type {import('eslint').Linter.FlatConfig[]} */
+export default [
+	js.configs.recommended,
+	...ts.configs.recommended,
+	...svelte.configs['flat/recommended'],
+	prettier,
+	...svelte.configs['flat/prettier'],
+	{
+		languageOptions: {
+			globals: {
+				...globals.browser,
+				...globals.node
+			}
+		}
+	},
+	{
+		files: ['**/*.svelte'],
+		languageOptions: {
+			parserOptions: {
+				parser: ts.parser
+			}
+		}
+	},
+	{
+		ignores: ['build/', '.svelte-kit/', 'package/', 'vite.config.[jt]s.timestamp-*']
```

Comment:
I personally would remove the vite timestamp files here and and in the other ones as well. They're very rarely created when Vite crashes or is killed or something and then I just delete them. I don't think we need to clutter these files worrying about them
```suggestion
		ignores: ['build/', '.svelte-kit/', 'package/']
```